index.html

```diff
@@ -222,33 +226,7 @@ <h2 class="text-primary text-2xl font-medium">Featured Speakers</h2>
         <div
           id="speakers"
           class="flex mx-4 flex-col gap-8 box-border md:grid md:grid-cols-2 md:gap-x-4 md:gap-y-8 md:mx-0"
```

Comment:
- [ ] Great job implementing the changes requested, as mentioned by the previous reviewer https://github.com/yusufobr/conference-capstone/pull/2#discussion_r1138575601 please also fix the alignment of name and description to match the original design currently there are misaligned :point_down: 
![image](https://user-images.githubusercontent.com/97521557/225691903-1336a5fd-ca40-4f1a-af71-8cc2ba438738.png)

how they should be :point_down: 
![image](https://user-images.githubusercontent.com/97521557/225692209-6cd6e86c-0cf4-40c7-aa42-e4491b70f662.png)

you can try to reduce the gaps, and use flex or grid for better alignment :+1:

Reply:
done